### PR TITLE
Add Kindle Previewer cask

### PR DESCRIPTION
**Pull request by ericmarkmartin**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

**Comment by commitay** [comment]
`kindle-previewer` has been removed previously.

https://github.com/caskroom/homebrew-cask/pull/32076

**Comment by ericmarkmartin** [comment]
I saw that change. i CNR on my machine though I assumed maybe changes to the software on Amazon's side had fixed that.

**Comment by commitay** [comment]
Please remove the extra commits from this PR.

**Comment by ericmarkmartin** [comment]
Ah, I accidentally based off of the wrong branch. Will do.

**Comment by ericmarkmartin** [comment]
Done

File: Casks/kindle-previewer.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+cask 'kindle-previewer' do
+  version :latest
+  sha256 :no_check
+
+  # s3.amazonaws.com/kindlepreviewer3 was verified as official when first introduced to the cask
+  url 'https://s3.amazonaws.com/kindlepreviewer3/KindlePreviewerInstaller.pkg'
+  name 'Kindle Previewer 3'
+  homepage 'https://www.amazon.com/gp/feature.html'
```
Comment (commitay) [inline Casks/kindle-previewer.rb]: This URL gives a "Sorry we can't find that page" message

Comment (ericmarkmartin) [reply Casks/kindle-previewer.rb]: The homepage url?

**Comment by commitay** [comment]
The app seems to work okay, guess Amazon must have fixed it.

File: Casks/kindle-previewer.rb (hunk 1 above)
Comment (ericmarkmartin) [reply Casks/kindle-previewer.rb]: Should be fixed

**Comment by ericmarkmartin** [comment]
When I look at the file history on Github I also see the history of the deleted kindle-previewer.rb cask. Is it worth editing my first commit on this PR to reflect that it is a re-addition so as to avoid future confusion?

**Review by commitay — APPROVED** [review]

**Pull request merged**